svc-events/events/publishevttosubscriber.go

```diff
@@ -310,6 +312,7 @@ func (e *ExternalInterfaces) postEvent(destination, eventUniqueID string, event
 		go e.checkUndeliveredEvents(destination)
 		return
 	}
+	fmt.Println("Error while send ", err)
```

Comment:
is it required? can we move this to debug log

Reply:
done